### DISCUSSION TITLE
VOSK modules not working

### DISCUSSION THREAD

**Discussion by mushie4282** [OpenVoiceOS/ovos-buildroot#248] [Q&A]
Has anyone noticed that ovos-ww-plugin-vosk is not loading any wakeword or hotwords? 
I discovered this when checking my voice logs. I did reinstall vosk and rebooted, but that didn't fix the issue. Am I missing something?

**Comment by JarbasAl** [comment]
what errors are you having exactly and whats your ww config, we have several vosk users so the plugin should be functional

**Reply by mushie4282** [reply]
When I view voice.log, I don't see any vosk hotwords being loaded. Only those that use precise-lite or pocketsphinx. I'm currently running more test to pinpoint my issue.

**Comment by mushie4282** [comment]
I found out why VOSK is not reocogizing my wake word. I used `ovos-dinkum-listener` to see any logs and this popped up.
`WARNING (VoskAPI:UpdateGrammarFst():recognizer.cc:308) Ignoring word missing in vocabulary: 'Emergency'`

**Reply by mushie4282** [reply]
Before that warning sign, the logs read that it downloaded the default model.
```
LOG (VoskAPI:ReadDataFiles():model.cc:213) Decoding params beam=10 max-active=3000 lattice-beam=2
LOG (VoskAPI:ReadDataFiles():model.cc:216) Silence phones 1:2:3:4:5:6:7:8:9:10
LOG (VoskAPI:RemoveOrphanNodes():nnet-nnet.cc:948) Removed 0 orphan nodes.
LOG (VoskAPI:RemoveOrphanComponents():nnet-nnet.cc:847) Removing 0 orphan components.
LOG (VoskAPI:ReadDataFiles():model.cc:248) Loading i-vector extractor from /home/ovos/.local/share/vosk/vosk-model-small-en-us-0/ivector/final.ie
LOG (VoskAPI:ComputeDerivedVars():ivector-extractor.cc:183) Computing derived variables for iVector extractor
LOG (VoskAPI:ComputeDerivedVars():ivector-extractor.cc:204) Done.
LOG (VoskAPI:ReadDataFiles():model.cc:282) Loading HCL and G from /home/ovos/.local/share/vosk/vosk-model-small-en-us-0/graph/HCLr.fst /home/ovos/.local/share/vosk/vosk-model-small-en-us-0/graph/Gr.fst
LOG (VoskAPI:ReadDataFiles():model.cc:308) Loading winfo /home/ovos/.local/share/vosk/vosk-model-small-en-us-0/graph/phones/word_boundary.int
LOG (VoskAPI:UpdateGrammarFst():recognizer.cc:287) ["Emergency", "[unk]"] 
```

**Comment by mushie4282** [comment]
I fixed my previous warning message. I did not enter the config correctly. I had "sample": instead or "samples":
